packages/gravity-ui-web/src/sass/05-components/02-molecules/02-lists/01-two-column-list.scss

```diff
@@ -0,0 +1,26 @@
+.grav-c-two-column-list {
```

Comment:
Hmm.. this doesn't seem much different to the existing `.grav-c-two-columns-block`, which also lays out items in 2 columns using flexbox.

My preference would be to merge the two. That might also be a good time to rename the class to an "object" - e.g. `.grav-o-two-columns` or similar - since it's just a layout thing, not really a _component_.

On the Nunjucks side, we'd then also need to replace the 2 templates with one. The current `molecules-two-columns-block` one is a bit dumb TBH because it hard-codes location cards for the items.

A better approach would be for a `molecules-two-columns` Nunjucks template to define a `{% block %}` that can be extended elsewhere - such as our sample page templates - to contain anything we like.

Come to think of it, maybe we should combine `.grav-c-list-cards-basic` with this too! In terms of layout these are all doing more or less the same. And the feature of stretching an item to be full-width could well be useful beyond cards. Although, for now I'd be content for the two-column ones to be combined and we can look at merging in the cards list later.

Reply:
The only difference is the required list styles on the list (such as `list-style: none;`), so I can understand the want to merge them. They do behave slightly different (which might not be a good thing) layout wise.

I'm not sure about combining the behaviour `.grav-c-list-cards-basic` has with all two column layouts is desired but I can understand the potential uses of a more generic class to do that. So having a two column class and a separate two column with full width 'hanging' class makes sense to me.

Perhaps there should be two basic two-column classes too for the different content arrangements:
```
|0|1|    |0|3|
|2|3| or |1|4|
|4|5|    |2|5|
```

Reply:
Sounds sensible.

Will you be in tomorrow? Perhaps we can sit down and hash this out together?

Reply:
Changes made to create a two column object level helper class and refactor other components (or remove some). Feel free to re-review @james-nash  (i'm about to update the PR description).